core/renderers/thrasos/info.js

```diff
@@ -89,14 +90,14 @@ Blockly.thrasos.RenderInfo.prototype.shouldStartNewRow_ = function(input, lastIn
 Blockly.thrasos.RenderInfo.prototype.getInRowSpacing_ = function(prev, next) {
   if (!prev) {
     // Between an editable field and the beginning of the row.
-    if (next.isField() && next.isEditable) {
+    if (next && Blockly.blockRendering.Types.isField(next) && next.isEditable) {
       return this.constants_.MEDIUM_PADDING;
     }
     // Inline input at the beginning of the row.
-    if (next.isInput && next.isInlineInput()) {
+    if (next && next.isInput && Blockly.blockRendering.Types.isInlineInput(next)) {
```

Comment:
Do we need the isInput check if the next check is against isInlineInput?

Reply:
I'm not sure, I left the code as is without any functional changes. I only added some checks for nullability where the closure compiler was complaining.